messages_zones.go

```diff
@@ -36,7 +36,7 @@ type ZoneOwner struct {
 type ZoneMeta struct {
 	Step                    int    `json:"step,omitempty"`
 	PageRuleQuota           string `json:"page_rule_quota,omitempty"`
-	CustomCertificateQuota  int    `json:"custom_certificate_quota,omitempty"`
+	CustomCertificateQuota  string `json:"custom_certificate_quota,omitempty"`
```

Comment:
I have never used this field but in [docs](https://api.cloudflare.com/#zone-properties) it is int.